pypdf/_xobj_image_helpers.py

```diff
@@ -29,9 +29,9 @@
 
 
 try:
-    from PIL import Image
-except ImportError:
-    raise ImportError(
+    from PIL import Image, UnidentifiedImageError  # noqa: F401
+except ImportError:  # deprecated
```

Comment:
Why deprecate usage without *Pillow* without further notice and/or discussion?

Reply:
the comments is also interpreted as code check disactivation. You prefer a `# pragma: no cover` ?

Reply:
`deprecated` surely is wrong here, thus the pragma looks better.